packages/token-search-discovery-controller/src/token-search-discovery-controller.ts

```diff
@@ -0,0 +1,122 @@
+import type { RestrictedControllerMessenger } from '@metamask/base-controller';
+import { BaseController } from '@metamask/base-controller';
+import type { TokenSearchResponseItem } from './types';
+
+export const TOKENSEARCH_EVENTS = {
+  SEARCH_COMPLETED: 'SEARCH_COMPLETED',
+} as const;
+
+const name = 'TokenSearchDiscoveryController';
```

Comment:
Can we call this `controllerName` for clarity?

``` suggestion
const controllerName = 'TokenSearchDiscoveryController';
```

Reply:
100%